api/src/main/java/ai/djl/modality/cv/translator/YoloV5Translator.java

```diff
@@ -0,0 +1,271 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package ai.djl.modality.cv.translator;
+
+import java.util.*;
+import java.awt.geom.Rectangle2D;
+import java.awt.geom.Rectangle2D.Double;
+import ai.djl.modality.cv.output.*;
+import ai.djl.ndarray.*;
+import ai.djl.ndarray.types.DataType;
+import ai.djl.translate.*;
+
+/**
+ A translator for yolo V5 models. This was tested with ONNX exported Yolo models.
+ For details check here: https://github.com/ultralytics/yolov5
+ */
+public class YoloV5Translator extends ObjectDetectionTranslator {
+   private final YOLO_OUTPUT_TYPE yoloOutputLayerType;
+   private double nmsThresh = 0.4F;
+
+   public enum YOLO_OUTPUT_TYPE {
```

Comment:
1. enum name should be: YolyOuputType
2. class should be at the end of the class,